redbot/cogs/mod/settings.py

```diff
@@ -199,3 +204,20 @@ class ModSettings(MixinMeta):
                     command=f"{ctx.prefix}unban"
                 )
             )
+
+    @modset.command()
+    @commands.guild_only()
+    async def toggledm(self, ctx: commands.Context, enabled: bool = None):
+        """Toggle whether to send a message to a user when they are 
+        kicked/banned.
+        If this is True, the bot will attempt to DM the user with the guild name
```

Comment:
This needs another new line to show up properly in help.
\+ Rephrased a little to make it more obvious to user.
```suggestion

        If this option is enabled, the bot will attempt to DM the user with the guild name
```